packages/lit-dev-content/site/docs/v3/releases/upgrade.md

```diff
@@ -1,40 +1,63 @@
 ---
-title: Upgrade guide
+title: Upgrade to Lit 3.x guide
 eleventyNavigation:
-  key: Upgrade guide
+  key: Upgrade to Lit 3.x guide
   parent: Releases
   order: 3
 versionLinks:
   v2: releases/upgrade/
 ---
 
-## Overview
+<div class="alert alert-info">
+
+If you are looking to migrate from Lit 1.x to Lit 2.x, see the [Lit 2.0 upgrade guide](/docs/v2/releases/upgrade/).
```

Comment:
similar to above, can we drop the .x and call it "Lit 2 upgrade guide"?

Reply:
Yes!